re-frisk/src/re_frisk/ui.cljs

```diff
@@ -56,19 +56,30 @@
     (fn []
       (when-not @ext-opened?
         (let [left (or (utils/normalize-draggable (:x @drag/draggable))
-                       (- js/window.innerWidth 30))]
+                       (- js/window.innerWidth 30))
+              handle-toggle (fn [] 
+                              (when-not (utils/closed? left)
+                                     (swap! db/tool-state assoc :latest-left (- js/window.innerWidth left)))
+                              (swap! drag/draggable assoc :x (- js/window.innerWidth
+                                                                (if (utils/closed? left) @latest-left 30))))
+
+              handle-keydown (fn [e]
+                               (let [input-elements #{"INPUT" "SELECT" "TEXTAREA"}
+                                     input-focused? (contains? input-elements  (.-tagName (.-target e)))]
+                                 (when (and (not input-focused?)
+                                            (= (.-key e) "h")
+                                            (.-ctrlKey e))
+                                   (.preventDefault e)
+                                   (handle-toggle))))
+              _ (js/window.addEventListener "keydown" handle-keydown)]
```

Comment:
it seems this listener might be added several times

Reply:
Good point, I'll see if I can fix it in a bit.

Reply:
the simplest way would be to add `_ (js/window.removeEventListener "keydown" handle-keydown)`.  before, its not  perfect, but fine